srv/components/csm/files/ha-params.tmpl

```diff
@@ -5,5 +5,8 @@
 {%- endif -%}
 interface: {{ mgmt_if }}
 vip: {{ pillar['cluster']['mgmt_vip'] }}
+{% set netmask = pillar['cluster'][grains['id']]['network']['mgmt_nw']['netmask'] %}
+{% set cidr = salt.network.calc_net(pillar['cluster']['mgmt_vip'], netmask) %}
```

Comment:
@dmitrydb 
Could you try using:
```
(salt['network.calc_net'](pillar['cluster']['mgmt_vip'], netmask)).split("/")[1]
```
It would be a preferred way for template/sls files

Reply:
Updated template

Reply:
Thanks